src/api/Feed.js

```diff
@@ -1178,6 +1180,58 @@ class Feed extends Base {
         return pendingFeedItem;
     };
 
+    /**
+     * Add a placeholder pending comment (reply).
+     *
+     * @param {string} parentId - id of parent comment or annotation
+     * @param {Object} currentUser - the user who performed the action
+     * @param {Object} commentBase - Base properties for reply (comment) to be added to the feed as pending.
+     * @return {Comment} - newly created pending reply
+     */
+    addPendingReply = (parentId: string, currentUser: User, commentBase: Object): Comment => {
+        if (!this.file.id) {
+            throw getBadItemError();
+        }
+        if (!currentUser) {
+            throw getBadUserError();
+        }
+
+        const date = new Date().toISOString();
+        const pendingReply: Comment = {
+            created_at: date,
+            created_by: currentUser,
+            modified_at: date,
+            isPending: true,
+            ...commentBase,
+        };
+
+        const getItemWithPendingReply = <T: { replies?: Array<Comment> }>(item: T, reply: Comment): T => {
```

Comment:
Does this need to be inlined in `addPendingReply`?

Reply:
It is used only in `addPendingReply` method and its purpose is nothing more then be a helper function for this method. Do you think I should move it outside of the Class?